playbook/app/pb_kits/playbook/pb_button/_button_mixins.scss

```diff
@@ -86,6 +88,10 @@ $pb_button_border_width: 0px;
     &:hover {
       @include pb_button_hover;
     }
+    &:active {
+      transition: none;
```

Comment:
@Israel-Molestina Can we not add transitions? I was going to suggest adding transitions as the different state switched are jarring. See how smooth the states (including focus) is on the [checkbox kit](https://playbook.powerapp.cloud/kits/checkbox/react)

Reply:
@jasoncypret I've added transition to the focus states. But when I talked to Ramon he did not want transitions on the active states. This way it is very obvious that you just clicked the button